Example/Tests/GINIImageAnalysisNoResultsViewControllerTests.swift

```diff
@@ -0,0 +1,140 @@
+        XCTAssertEqual(3, suggestionsCollectionItemsCount, "suggestionsCollectionView items count should be equal to the mocked items count declared above")
+    }
+    
+    func testSuggestionCollectionItemsCount(){
+        let suggestionsCount = viewController.captureSuggestions.count
+        
+        let suggestionsCollectionItemsCount = viewController.collectionView(viewController.suggestionsCollectionView, numberOfItemsInSection: 0)
+        
+        XCTAssertEqual(suggestionsCount, suggestionsCollectionItemsCount, "suggestionsCollectionView items count should be equal to captureSuggestions array count")
+    }
+    
+    func testFirstMockedSuggestionCollectionCellText() {
+        viewController.captureSuggestions = mockedItems
+        let suggestionCollectionCell = viewController.collectionView(viewController.suggestionsCollectionView, cellForItemAt: IndexPath(row: 0, section: 0)) as! CaptureSuggestionsCollectionCell
+        
+        let suggestionCollectionCellText = suggestionCollectionCell.suggestionText.text
+        
+        XCTAssertEqual("item 1 text", suggestionCollectionCellText, "first suggestionsCollectionView item text should be equal to first mocked item text")
+    }
+    
+    func testFirstSuggestionCell() {
+        let indexPath = IndexPath(row: 0, section: 0)
+        
+        let suggestion1Text = viewController.captureSuggestions[indexPath.row].text
+        let suggestion1Image = viewController.captureSuggestions[indexPath.row].image
+        
+        let suggestionCollectionCell = viewController.collectionView(viewController.suggestionsCollectionView, cellForItemAt: indexPath) as! CaptureSuggestionsCollectionCell
+        
+        XCTAssertEqual(suggestion1Text, suggestionCollectionCell.suggestionText.text, "first suggestionsCollectionView item text should be equal to first captureSuggestions item text")
+        XCTAssertEqual(suggestion1Image, suggestionCollectionCell.suggestionImage.image, "first suggestionsCollectionView item image should be equal to first captureSuggestions item image")
+
+    }
+    
+    func testSecondSuggestionCell() {
+        let indexPath = IndexPath(row: 1, section: 0)
+        
+        let suggestion2Text = viewController.captureSuggestions[indexPath.row].text
+        let suggestion2Image = viewController.captureSuggestions[indexPath.row].image
+        
+        let suggestionCollectionCell = viewController.collectionView(viewController.suggestionsCollectionView, cellForItemAt: indexPath) as! CaptureSuggestionsCollectionCell
+        
+        XCTAssertEqual(suggestion2Text, suggestionCollectionCell.suggestionText.text, "second suggestionsCollectionView item text should be equal to second captureSuggestions item text")
+        XCTAssertEqual(suggestion2Image, suggestionCollectionCell.suggestionImage.image, "second suggestionsCollectionView item image should be equal to second captureSuggestions item image")
+        
+    }
+    
+    func testThirdSuggestionCell() {
```

Comment:
Same as the other pull request - you don't have to test each cell since the code being run is the same.

Or alternatively, test against the real production data that's going to be shown at runtime.

Reply:
I answered you in the other PR with the reason.